### PR TITLE
Adding Some Information about how to get the OpenAI API Token

### PR DESCRIPTION
**Pull request by KaueSabinoSRV17**
- docs: explaining how to get your openai api token
- docs: source the .bashrc file after changes

I started using the CLI yesterday, and it is awesome! I am very happy to help out with anything i can

### CODE REVIEW

**Comment by KaueSabinoSRV17** [comment]
So, about the PR, i had a little bit of trouble to find where i could get the API Token, so i thought it would be welcome to explain that on the README file.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ cd gpt-cli
 pip install -r requirements.txt
 ```
 
+To use this CLI, you need to have an OpenAI API token, and add it as an Enviroment Variable on your
+Operating System.
```
Comment (kharvd) [inline README.md:31]:
```suggestion
operating system.
```

File: README.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ cd gpt-cli
 pip install -r requirements.txt
 ```
 
+To use this CLI, you need to have an OpenAI API token, and add it as an Enviroment Variable on your
```
Comment (kharvd) [inline README.md:30]:
```suggestion
To use this CLI, you need to have an OpenAI API token, and add it as an environment variable on your
```

File: README.md [hunk 3]
```diff
@@ -27,6 +27,11 @@ cd gpt-cli
...
+Operating System.
+
+You can get you OpenAI API token on the [OpenAI Plataform Page](https://platform.openai.com/account/api-keys)
```
Comment (kharvd) [inline README.md:33]:
```suggestion
You can get you OpenAI API token on the [OpenAI Platform Page](https://platform.openai.com/account/api-keys)
```

File: README.md [hunk 4]
```diff
@@ -35,6 +40,11 @@ nano ~/.bashrc
 export OPENAI_API_KEY=<your_key_here>
 ```
 
+After that, to make the changes to the `.basrc` file take effect without opening another terminal,
```
Comment (kharvd) [inline README.md:43]:
```suggestion
After that, to make the changes to the `.bashrc` file take effect without opening another terminal,
```

**Review by kharvd — CHANGES_REQUESTED** [review]
Thanks! There are a few typos, let's fix before merging

**Comment by KaueSabinoSRV17** [comment]
Hi, i fixed the typos, it should be alright now!

**Comment by KaueSabinoSRV17** [comment]
Hello, i think you requested one more change, but i can't see what it is.

**Review by kharvd — APPROVED** [review]

**Comment by KaueSabinoSRV17** [comment]
Hi, sorry, i saw that the changes where aproved but not merged, and thought that i should close the PR to merge the changes.

File: README.md [hunk 5]
```diff
@@ -35,6 +39,11 @@ nano ~/.bashrc
 export OPENAI_API_KEY=<your_key_here>
 ```
 
+After that, to make the changes to the `.bashrc` file take effect without opening another terminal,
+you can run:
+
+    source ~/.bashrc
```
Comment (AlexanderYastrebov) [inline README.md:45]: I think it makes sense to use code block here for consistency with the rest of the file

Comment (KaueSabinoSRV17) [reply README.md:45]: Hello

Ok, just to be sure, you mean a ```bash``` block right?

Comment (AlexanderYastrebov) [reply README.md:45]: yes

**Comment by KaueSabinoSRV17** [comment]
I've added `bash` blocks not just on my addition, but in the other parts of the README as well.

File: README.md [hunk 6]
```diff
@@ -47,7 +58,7 @@ You can also use a `.gptrc` file for configuration. See the [Configuration](READ
 
 Make sure to set the `OPENAI_API_KEY` environment variable to your OpenAI API key (or put it in the `~/.gptrc` file as described below).
 
-```
+```bash
```
Comment (kharvd) [inline README.md:61]: This is not a bash code block, could you please revert this line?

File: README.md [hunk 7]
```diff
@@ -94,7 +105,7 @@ To enter multi-line mode, enter a backslash `\` followed by a new line. Exit the
 
 You can override the model parameters using `--model`, `--temperature` and `--top_p` arguments at the end of your prompt. For example:
 
-```
+```bash
```
Comment (kharvd) [inline README.md:108]: This is not bash either

**Review by kharvd — CHANGES_REQUESTED** [review]
Sorry for the delay, a couple of more suggestions before I merge

**Pull request closed without merging**